roles/haproxy/templates/keepalived.conf.j2

```diff
@@ -24,17 +24,26 @@ vrrp_script check_haproxy {
 }
 
 vrrp_instance VI_1 {
-    state {% if ansible_hostname.endswith('-1') %}MASTER{% else %}BACKUP{% endif %}
+    state {% if ansible_hostname.endswith('1') %}MASTER{% else %}BACKUP{% endif %}
     interface {{ keepalived_interface }}
     virtual_router_id {{ keepalived_virtual_router_id }}
-    priority {% if ansible_hostname.endswith('-1') %}101{% else %}100{% endif %}
+    priority {% if ansible_hostname.endswith('1') %}101{% else %}100{% endif %}
+
+    unicast_peer {
+        {% for host in play_hosts %}
```

Comment:
`play_hosts` seems to be deprecated:
https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html#term-play_hosts
```suggestion
        {% for host in ansible_play_batch %}
```